samples/endpoint_resouces_samples.py

```diff
@@ -130,7 +131,8 @@ def _check_stor(storage_name_or_id):
         _check_stor(default)
 
     # check backend mappings:
-    for mapped_storage in storage_mappings.get('backend_mappings', {}).values():
+    for mapped_storage in (storage_mappings.get('backend_mappings', {}).
+                           values()):
```

Comment:
Let's rather put this into a variable
```suggestion
    backend_mappings = storage_mappings.get('backend_mappings', {})
    for mapped_storage in backend_mappings.values():
```